mlflow/utils/databricks_utils.py

```diff
@@ -171,7 +171,19 @@ def is_in_databricks_job():
 
 
 def is_in_databricks_model_serving_environment():
-    return "IS_IN_DATABRICKS_MODEL_SERVING_ENV" in os.environ
+    """
+    Check if the code is running in Databricks Model Serving environment.
+    The environment variable set by Databricks when starting the serving container.
+    """
+    val = (
+        os.environ.get("IS_IN_DB_MODEL_SERVING_ENV")
+        # Checking the old env var name for backward compatibility. The env var was renamed once
+        # to fix a model loading issue, but we still need to support it for a while.
+        # TODO: Remove this once the new env var is fully rolled out.
```

Comment:
do we have internal ticket to track the TODO (as a task reminder )

Reply:
I filed a cleanup ticket for this and the safe flag special handling for the legacy env var naming

Reply:
Created ML-41733 and linked to the original ticket!